record/record.app.js

```diff
@@ -163,10 +164,11 @@
                 var recordSetLink = page.reference.contextualize.compact.appLink;
 
                 if (page.tuples.length < 1) {
-                    throw new Errors.noRecordError(context.filter.filters, recordSetLink);
+                    throw new Errors.noRecordError({}, recordSetLink);
```

Comment:
I think it's better if we use the unfiltered url (`page.reference.unfiltered.contextualize.compact.appLink`) in this case for going to recordset. Currently it will go to a recordset page that has no data in it.